### PR TITLE
WIP - Objects Storage Adapter

### PR DESCRIPTION
**Pull request by gabrielwas**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/ObjectsDDMStorageAdapter.java [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+	immediate = true, property = "ddm.storage.adapter.type=objects",
+	service = DDMStorageAdapter.class
+)
+public class ObjectsDDMStorageAdapter implements DDMStorageAdapter {
+
+	@Override
+	public DDMStorageAdapterDeleteResponse delete(
+		DDMStorageAdapterDeleteRequest ddmStorageAdapterDeleteRequest)
+		throws StorageException {
+		return null;
```
Comment (thiago-farias) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/ObjectsDDMStorageAdapter.java]: If it's gonna be temporarily not implemented, it's a good practice throwing an exception:
`throw new UnsupportedOperationException("Not supported yet.");`

Comment (gabrielwas) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/ObjectsDDMStorageAdapter.java]: Good suggestion, Thiago. Thx!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/ObjectsDDMStorageAdapter.java [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+		}
+		catch (PortalException e) {
+			e.printStackTrace();
+		}
+
+		return null;
+	}
+
+	public static String capitalize(String str)
```
Comment (thiago-farias) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/ObjectsDDMStorageAdapter.java]: Please, use `StringUtil::upperCaseFirstLetter` instead.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/ObjectsDDMStorageAdapter.java (hunk 1 above)
Comment (gabrielwas) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/ObjectsDDMStorageAdapter.java]: Done! :D

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/ObjectsDDMStorageAdapter.java (hunk 2 above)
Comment (gabrielwas) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/storage/ObjectsDDMStorageAdapter.java]: Done! :D

**Pull request closed without merging**